orchagent/copporch.cpp

```diff
@@ -18,8 +24,11 @@ extern sai_switch_api_t*    sai_switch_api;
 
 extern sai_object_id_t      gSwitchId;
 extern PortsOrch*           gPortsOrch;
+extern Directory<Orch*>     gDirectory;
 extern bool                 gIsNatSupported;
 
+#define FLEX_COUNTER_UPD_INTERVAL 1
```

Comment:
Please change this to 10.

Reply:
This interval here is the timer to add flex counter to flex DB, it is not the query interval.

Reply:
But this is the interval where the select gets triggered, right? So why select this every 1 sec?

Reply:
Here we use a timer to timely add flex counter to FLEX COUNTER DB due to comment: [#1859 (comment)](https://github.com/Azure/sonic-swss/pull/1859#discussion_r727988279) . We set the interval 1 here because we want flex counter to be created as soon as possible. It won't cost much CPU because:

- the timer will be started when the first item is added to queue "m_pendingAddToFlexCntr"
- when the queue "m_pendingAddToFlexCntr" is empty, the timer will be stopped

Please check copporch.cpp line 770:
```cpp
    auto was_empty = m_pendingAddToFlexCntr.empty();
    m_pendingAddToFlexCntr[counter_id] = trap_name;

    if (was_empty)
    {
        m_FlexCounterUpdTimer->start();
    }
```
and copporch.cpp line 1229

```cpp
    if (m_pendingAddToFlexCntr.empty())
    {
        m_FlexCounterUpdTimer->stop();
    }
```